webapp/tests/forms/steps/lotse/test_pauschbetrag.py

```diff
@@ -184,6 +190,73 @@ def test_if_merkzeichen_h_and_no_pflegegrad_set_then_return_7400_for_all_disabil
             calculated_pauschbetrag = calculate_pauschbetrag(**params, disability_degree=disability_degree)
             assert calculated_pauschbetrag == 7400
 
+    def test_if_no_parameters_then_return_zero(self):
+        calculated_pauschbetrag = calculate_pauschbetrag()
+        assert calculated_pauschbetrag == 0
+        
+    def test_if_disability_degree_under_20_then_return_zero(self):
+        calculated_pauschbetrag = calculate_pauschbetrag(disability_degree=19)
+        assert calculated_pauschbetrag == 0
```

Comment:
I am wondering if we might need to test this also with the other params (has_pflegegrad etc.).